src/views/ajax.js

```diff
@@ -5,6 +5,36 @@ const arrowIcon = require('./open').arrowIcon;
 const icons = require('../assets/icons').default;
 const statusIcon = require('../assets/icons').statusIcon;
 
+/**
+ * removeOrigin - function to remove origin from a URL.
+ *
+ * @param {String} url URL to remove the origin from.
+ * @param {String} origin Origin to remove from URL.
+ */
+const removeOrigin = (url = '', origin = window.location.origin) => {
```

Comment:
We'll make tests for the HUD once it's refactored. It's not exactly in a test-friendly state right now.